lib/manager/gradle-wrapper/artifacts.ts

```diff
@@ -0,0 +1,69 @@
+import Git from 'simple-git/promise';
+import { resolve } from 'path';
+import { logger } from '../../logger';
+import { UpdateArtifact, UpdateArtifactsResult } from '../common';
+import { exec } from '../../util/exec';
+import { readLocalFile } from '../../util/fs';
+import { platform } from '../../platform';
+import { VERSION_REGEX } from './search';
+
+const gradlewFilename =
+  process.platform === 'win32' ? 'gradlew.bat' : 'gradlew';
+
+async function addIfUpdated(
+  status: Git.StatusResult,
+  filePath: string
+): Promise<UpdateArtifactsResult | null> {
+  if (status.modified.includes(filePath)) {
+    return {
+      artifactError: null,
+      file: {
+        name: filePath,
+        contents: await readLocalFile(filePath),
+      },
+    };
+  }
+  return null;
+}
+
+export async function updateArtifacts({
+  packageFileName,
+  updatedDeps,
+  newPackageFileContent,
+  config,
+}: UpdateArtifact): Promise<UpdateArtifactsResult[] | null> {
+  try {
+    const projectDir = resolve(packageFileName, './../../../');
+    logger.debug(updatedDeps, 'gradle-wrapper.updateArtifacts()');
+    const gradlewPath = resolve(projectDir, `./${gradlewFilename}`);
+    const version = VERSION_REGEX.exec(newPackageFileContent).groups.version;
```

Comment:
Isn't the version already available in `updatedDeps`?

Reply:
AFAIK, the `updatedDeps` contains only the names of the updated dependencies.